build/rule-generator/directories.js

```diff
@@ -0,0 +1,22 @@
+/* global Promise */
+const path = require('path');
+
+const directories = {
+	axePath: path.join(__dirname, '..', '..', 'axe.js'),
+	rules: path.join(__dirname, '..', '..', 'lib', 'rules'),
+	checks: path.join(__dirname, '..', '..', 'lib', 'checks'),
+	testRuleMatches: path.join(__dirname, '..', '..', 'test', 'rule-matches'),
+	testChecksUnit: path.join(__dirname, '..', '..', 'test', 'checks'),
+	testChecksIntegration: path.join(
+		__dirname,
+		'..',
+		'..',
+		'test',
+		'integration',
+		'rules'
+	)
+};
+
+module.exports = {
```

Comment:
What else do you expect to have in this module? If all you're going to ever export is an object, it seems unnecessary to nest it within another.

Also, `const directories = require('./directories).directories` is pretty redundant.

I suggest changing this to `module.exports = directories` or exporting directories individually (`exports.testChecksIntegration = path.join(....)`).